scripts/generate_climos

```diff
@@ -18,7 +18,8 @@ def main(args):
             except Exception as e:
                 logger.info('{}: {}'.format(e.__class__.__name__, e))
             else:
-                logger.info('climo_periods: {}'.format(input_file.climo_periods.keys()))
+                periods = input_file.climo_periods.keys() & args.climo
+                logger.info('climo_periods: {}'.format(periods))
```

Comment:
Isn't `args.climo` a list? The `&` operator isn't defined when used in an expression with types `dict_keys` and `list` respectively. It doesn't like `list`, though it does like `dict_keys`.